.eslintrc.json

```diff
@@ -4,7 +4,8 @@
     "plugin:node/recommended",
```

Comment:
Would it make sense to remove this configuration for the `node` plugin as you've removed the plugin itself below?

Reply:
Great question! It is used, and is included via this line that has been commented :) the `node:recommended` config has
```
plugins: ["node"]
```
"baked in"